packages/api-reference/src/components/Layouts/ModernLayout.vue

```diff
@@ -22,6 +23,14 @@ const config = computed(() => {
     : props.configuration?.showSidebar
   return { ...props.configuration, showSidebar }
 })
+
+const { activeItemId } = useNavigation()
+
+watch(activeItemId, () => {
```

Comment:
Might be could to do a quick comparison too

```suggestion
watch(activeItemId, (n,o) => {
    if (n && n !== o) {
```